resources/bin/orchestrator-client

```diff
@@ -233,6 +233,10 @@ function filter_keys {
   cat - | jq '.[] | .Key'
 }
 
+function filter_broken {
+  cat - | jq 'reduce .[] as $instance ([]; if $instance.LastSQLError != "\"\"" then . + [$instance] elif $instance.LastIOError != "\"\"" then . + [$instance] else . + [] end)'
```

Comment:
here's a shorter way to write this:

```suggestion
  cat - | jq '.[] | select(.LastSQLError != "" or .LastIOError != "")'
```

Reply:
Ah, I used `select` before but had a problem with something, but it was late and I probably mistyped something